src/SprykerEco/Yves/AmazonPay/Theme/default/payment/get-shipment-methods.twig

```diff
@@ -20,6 +23,10 @@
             );
         });
 
-        $('input[name=amazonpayShipmentMethod]:first').click();
+        if ($('input[name=amazonpayShipmentMethod]:checked').length === 0) {
```

Comment:
I'm not sure that it's allowed to write js code into twig templates. Please ask FE team

Reply:
There is no any sense to fix just this template bcs there is JS almost in every template. As for me we should refactor whole module.

Reply:
I will provide estimation for this implementation.